base/src/main/java/com/smartdevicelink/protocol/enums/ControlFrameTags.java

```diff
@@ -44,6 +44,7 @@
 	private static class NAKBase{
 		/** An array of rejected parameters related to the corresponding request*/
 		public static final String REJECTED_PARAMS = "rejectedParams";
+		public static final String REASON = "reason";
 	}
 
```

Comment:
The reason value for all NAKs messages should be printed as a warning similar to how we do it here:
https://github.com/smartdevicelink/sdl_java_suite/blob/7f12aef2f619380fc625cb72bfa1a1948d425181/base/src/main/java/com/smartdevicelink/protocol/SdlProtocolBase.java#L1428